docs/Install.md

```diff
@@ -6,22 +6,11 @@ Inductiva package is simple to install, just run on your terminal:
 pip install --upgrade inductiva
 ```
 
-This will provide the core functionalities of the API, which allows you to submit jobs, control machines and run simulations. To use the visualization and post-processing tools, you need to install additional optional dependencies specific to different scientific domains: `molecules_extra`, `fluids_extra` or `coastal_extra`. For example, for fluid dynamics:
-
-```
-pip install --upgrade "inductiva[fluids_extra]"
-```
-
+This will provide the core functionalities of the API, which allows you to submit jobs, control machines and run simulations. 
 For any issues with the installation follow the next steps.
 
 ## Installation troubleshooting
 
-### Why can't I install the optional packages?
-Depending on your shell, you may encounter issues when trying to install optional packages such as `inductiva[molecules_extra]`. This is because certain shells interpret brackets, like those in `[molecules_extra]`, in a special way. To prevent any misinterpretation or errors, enclose the package name and its extras in double quotes. To ensure a successful installation, please use the following command:
-
-```bash
-pip install --upgrade "inductiva[molecules_extra]"
-```
 
 ### Why can't I install Inductiva package?
```

Comment:
I think now you can remove this sub-header "Why can't I install Inductiva package", and leave only the main header "Installation troubleshooting"...

Reply:
on it!